src/DucksNet.API/DTO/CreateCageDTO.cs

```diff
@@ -0,0 +1,5 @@
+public class CreateCageDTO
+{
+    public string SizeString { get; set; } = string.Empty;
+    public Guid LocationId { get; set; }
```

Comment:
this can not be null?

Reply:
Nope. They are required to be sent by client.